app/models/mixins/relationship_mixin.rb

```diff
@@ -210,12 +225,16 @@ def ancestor_rels(*args)
   # Returns a list of ancestor records, starting with the root record and ending
   #   with the parent record
   def ancestors(*args)
+    return call_ancestry_method(:ancestors) if use_ancestry?
+
     Relationship.resources(ancestor_rels(*args))
   end
 
   # Returns a list of ancestor class/id pairs, starting with the root class/id
   #   and ending with the parent class/id
   def ancestor_ids(*args)
+    return send(:parse_ancestry_column, read_attribute(self.ancestry_base_class.ancestry_column)) if use_ancestry?
```

Comment:
weird...there's no ancestor_ids method in the ancestry gem?

Reply:
Additionally, this method used to return class/id pairs, but this method doesn't do that?  Do the callers care?

Reply:
> weird...there's no ancestor_ids method in the ancestry gem?

Don't worry, the gem has the method.

> Additionally, this method used to return class/id pairs, but this method doesn't do that? Do the callers care?

When we change this across, we will need to ensure that the classes using that method also change.

Aside: I often had to change the ids returned here to actually return the ids.

Reply:
can we just use the standard convention here?

```suggestion
return call_ancestry_method(:ancestor_ids) if use_ancestry?
```